project/sites/hiaredressni/themes/hiaredressni_theme/src/scss/init/_variables.scss

```diff
@@ -48,5 +48,4 @@ $max-grid-width: 1024px !default;
 
 $main-menu-border: false;
 
-$two-col-layout-sidebar-second: (65%, 30%);
```

Comment:
Was this meant to be removed from the HIA Redress site?

Reply:
No it was not , good spot! This is now corrected. Thanks